src/components/OpenwbBaseCard.vue

```diff
@@ -97,22 +97,33 @@ export default {
     collapsible: { type: Boolean, default: false },
     collapsed: { type: Boolean, default: false },
   },
+  emits: ["collapsed", "expanded"],
   data() {
     return {
       isCollapsed: this.collapsible && this.collapsed,
     };
   },
+  watch: {
+    collapsed(newVal) {
+      this.isCollapsed = newVal;
+    },
+  },
   methods: {
     toggleBody() {
       if (this.collapsible === true) {
         this.isCollapsed = !this.isCollapsed;
       }
+      if (this.isCollapsed) {
+        this.$emit("collapsed");
+      } else {
+        this.$emit("expanded");
+      }
```

Comment:
Es sollte nur ein Event generiert werden, wenn die Anzeige auch umgeschaltet wurde.
```suggestion
      if (this.collapsible === true) {
        this.isCollapsed = !this.isCollapsed;
        this.$emit(this.isCollapsed ? "collapsed" : "expanded");
      }
```

Reply:
ist angepasst.